homeassistant/components/waze_travel_time/config_flow.py

```diff
@@ -62,11 +62,11 @@ async def async_step_init(self, user_input=None) -> FlowResult:
                     vol.Optional(
                         CONF_INCL_FILTER,
                         default=self.config_entry.options.get(CONF_INCL_FILTER, ""),
-                    ): cv.string,
+                    ): vol.Maybe(cv.string),
```

Comment:
This can be resolved by making `DATA_SCHEMA` a constant, and then using `suggested_values`:
```python
DATA_SCHEMA = vol.Schema(
    {
        vol.Optional(
            CONF_INCL_FILTER, default="",
        ): cv.string,
    }
)
...
data_schema = self.add_suggested_values_to_schema(DATA_SCHEMA, self.config_entry.options)
return self.async_show_form(
    step_id="init",
    data_schema=data_schema
)
```